test/integration/scheduler/predicates_test.go

```diff
@@ -812,67 +813,68 @@ func TestInterPodAffinity(t *testing.T) {
 					NodeName:   nodes[0].Name}, ObjectMeta: metav1.ObjectMeta{Name: "fakename", Labels: map[string]string{"foo": "abc"}}},
 			},
 			fits: false,
-			test: "nodes[0] and nodes[1] have same topologyKey and label value. nodes[0] has an existing pod that matches the inter pod affinity rule. The new pod can not be scheduled onto either of the two nodes.",
 		},
 	}
 
 	for _, test := range tests {
-		for _, pod := range test.pods {
-			var nsName string
-			if pod.Namespace != "" {
-				nsName = pod.Namespace
-			} else {
-				nsName = testCtx.NS.Name
-			}
-			createdPod, err := cs.CoreV1().Pods(nsName).Create(context.TODO(), pod, metav1.CreateOptions{})
-			if err != nil {
-				t.Fatalf("Test Failed: error, %v, while creating pod during test: %v", err, test.test)
+		t.Run(test.name, func(t *testing.T) {
+			for _, pod := range test.pods {
+				var nsName string
+				if pod.Namespace != "" {
+					nsName = pod.Namespace
+				} else {
+					nsName = testCtx.NS.Name
+				}
+				createdPod, err := cs.CoreV1().Pods(nsName).Create(context.TODO(), pod, metav1.CreateOptions{})
+				if err != nil {
+					t.Fatalf("Test Failed: error, %v, while creating pod.", err)
```

Comment:
```suggestion
					t.Fatalf("Error while creating pod: %v", err)
```

is more conventional. (applies elsewhere)